app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/room/RoomRepresentativePicture.kt

```diff
@@ -0,0 +1,12 @@
+package com.github.HumanLearning2021.HumanLearningApp.room
+
+import androidx.room.Embedded
+import androidx.room.Relation
+
+data class RoomRepresentativePicture(
+    @Embedded val category: RoomCategory,
```

Comment:
Why is this embedded?

Reply:
Same reaseon as for https://github.com/HumanLearning2021/HumanLearningApp/pull/83#discussion_r612677346